### ISSUE TITLE
Is there any way to set flattener two-pass options?

### ISSUE THREAD

**Issue by ladm2110** [paolodragone/pymzn#32]
I want to pass to the compiler the option '-O5' to set a higher optimization level. I can't find a way to do it using pymzn. Am I missing something?

**Comment by paolodragone** [comment]
Hi!
Unfortunately I still haven't had the time to add support for the features of the newer versions of MiniZinc, the last one supported is the 2.1.7. I expect to be able to work my way through the new features by February. In the mean time, I suppose you could "cheat" your way around it by setting this option via the configuration of the path to the mzn2fzn binary:
```
import pymzn
pymzn.config.set('mzn2fzn', 'mzn2fzn -O5')
``` 
It should work, though I haven't tested it. Let me know if it works. I'll keep the issue open as a reminder.
Cheers!

**Comment by ladm2110** [comment]
Hello, I tried the way you proposed but it raises the following exception:
FileNotFoundError: [Errno 2] No such file or directory: 'mzn2fzn -O5': 'mzn2fzn -O5'

I found that if you add the line:
args += ['-O5']
inside the mzn2fzn method, I can accomplish that, but it feels wrong.

**Comment by paolodragone** [comment]
Yeah, sorry about that. OK, maybe another workaround (that I actually tested this time) would be:
```
import pymzn
pymzn.minizinc('your_file.mzn', '-O5')
```
Basically, you are pretending that '-O5' is a dzn file. In theory the function `minizinc` function expects only dzn files as additional positional arguments, but it basically gets anything really.
This is a bug, not a feature! :) As soon as I release the new version, that won't probably work anymore.
I'll also add some mechanism to ensure that if I miss some command line option or new ones get added to minizinc they will be easy to add without modifying the code.
Cheers!

**Comment by paolodragone** [comment]
Fixed in 0.18.1.
